.github/workflows/release-framework.yml

```diff
@@ -29,3 +29,54 @@ jobs:
           user: ${{ env.PYPI_USERNAME }}
           password: ${{ env.PYPI_PASSWORD }}
           packages-dir: ${{github.workspace}}/dist
+
+      - name: Deploy Stage
```

Comment:
Change this step name to something that is more descriptive - announce new version/send version announcement/send internal release notification/etc